packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -749,3 +749,24 @@ function convertToBasePropertyTypeUntagged(
 			return `INVALID PROPERTY (typed as ${typeof x})`;
 	}
 }
+
+export const tagData = <
+	T extends TelemetryDataTag,
+	V extends Record<string, TelemetryEventPropertyType>,
```

Comment:
Please use the `Ext` types in all APIs used for instrumenting.  The non-Ext ones like this should either be removed or used only with the "Base"-named types that are used across compat boundaries. (See AB#4545)

```suggestion
	V extends Record<string, TelemetryEventPropertyTypeExt>,
```